content/en/docs/tasks/configure-pod-container/configure-projected-volume-storage.md

```diff
@@ -67,6 +67,16 @@ the Pod:
     ```shell
     ls /projected-volume/
     ```
+
+## Clean up
+
+Delete the Pod and the Secrets:
+
+```shell
+kubectl delete pod test-projected-volume
+kubectl delete secret user pass
+```
```

Comment:
Is it worth nudging the reader to remove `username.txt` and `password.txt` as well?

Reply:
As @tengqm said, I also believe cleaning k8s resources is enough. People following this tutorial should have already the minimum of Linux knowledge to `rm` the created files.